test/checks/mobile/target-size.js

```diff
@@ -0,0 +1,106 @@
+describe('target-size tests', function () {
+  'use strict';
+
+  var checkContext = axe.testUtils.MockCheckContext();
+  var checkSetup = axe.testUtils.checkSetup;
+  var shadowCheckSetup = axe.testUtils.shadowCheckSetup;
+  var shadowSupport = axe.testUtils.shadowSupport.v1;
+  var isIE11 = axe.testUtils.isIE11;
+  var check = checks['target-size'];
+
+  afterEach(function () {
+    checkContext.reset();
+  });
+
+  // IE cannot count
+  (isIE11 ? xit : it)(
+    'returns false for targets smaller than minSpacing',
+    function () {
+      var checkArgs = checkSetup(
+        '<button id="target" style="' +
+          'display: inline-block; width:20px; height:30px;' +
+          '">x</button>'
+      );
+      assert.isFalse(check.evaluate.apply(checkContext, checkArgs));
+      assert.deepEqual(checkContext._data, {
+        minSpacing: 24,
+        width: 20,
+        height: 30
+      });
+    }
+  );
+
+  it('returns true for unobscured targets larger than minSpacing', function () {
+    var checkArgs = checkSetup(
+      '<button id="target" style="' +
+        'display: inline-block; width:40px; height:30px;' +
+        '">x</button>'
+    );
+    assert.isTrue(check.evaluate.apply(checkContext, checkArgs));
+    assert.deepEqual(checkContext._data, {
+      minSpacing: 24,
+      width: 40,
+      height: 30
+    });
+  });
+
+  it('returns true for obscured targets with insufficient space', function () {
```

Comment:
Should this be `sufficient space`? The test below this one has the same title but a different return.